README.md

```diff
@@ -22,21 +23,55 @@ py-irt is now available on PyPi!
 
 ### Pre-reqs
 
-1. Install [PyTorch](https://pytorch.org/get-started/locally/). 
-2. Install [Pyro](https://pyro.ai/) 
-3. Install py-irt: 
+1. Install [PyTorch](https://pytorch.org/get-started/locally/).
+2. Install [Pyro](https://pyro.ai/)
+3. Install py-irt:
+
+```shell
+pip install py-irt
+```
 
+OR
 
 ```shell
-pip install py-irt 
+git clone https://github.com/nd-ball/py-irt.git
+cd py-irt
+pip install -r requirements.txt
 ```
 
+## Usage
+
+Once you install from PyPI, you can use the following command to fit an IRT
+model on the scored predictions of a dataset. For example, if you were to run py-irt with the 4PL model on the scored predictions of different transformer models on the SQuAD dataset, you'd do this:
+`py-irt train 4pl ~/path/to/dataset/eg/squad.jsonlines /path/to/output/eg/test-4pl/`
+
+## FAQ
+
+1. What kind of output should I expect on running the command to train an IRT model?
+
+You should see something like this when you run the command given above:
+![image](https://user-images.githubusercontent.com/40918514/123986740-3eccd100-d9cf-11eb-8e58-ba5ad6c977ce.png)
+
+2. I tried installing py-irt using pip from PyPI. But when I try to run the command `py-irt train 4pl ~/path/to/dataset/eg/squad.jsonlines /path/to/output/eg/test-4pl/`, I get an error that says `bash: py-irt: command not found`. How do I fix this?
+
+You may be running into this error because of the old version on PyPI, which has not yet been configured for CLI usage. It's in the pipeline and you should be able to see it soon! In the meantime, do the following steps for replicating CLI function in the repo using these commands:
```

Comment:
Can you update this to say: "The CLI interface was implemented in PyPi version 0.2.1. If you are getting this error try updating py-irt:

```shell
pip install --upgrade py-irt
```

Alternatively, you can install the latest version from github: (include shell instructions from the below)

Reply:
Updated!